enterprise/frontend/src/metabase-enterprise/embedding/components/InteractiveEmbeddingSettings.tsx

```diff
@@ -36,25 +33,20 @@ export function InteractiveEmbeddingSettings() {
 
   return (
     <SettingsPageWrapper title={t`Interactive embedding`}>
-      <SettingsSection>
-        <EmbeddingToggle
-          settingKey="enable-embedding-interactive"
-          label={t`Enable interactive embedding`}
-        />
-
-        <Box>
-          <SettingHeader id="get-started" title={t`Get started`} />
-          <Button
-            mt="xs"
-            variant="outline"
-            component={ExternalLink}
-            href={quickStartUrl}
-          >{t`Check out the Quickstart`}</Button>
-        </Box>
+      <EmbeddingSettingsCard
+        title={t`Enable interactive embedding`}
+        titleProps={{ fz: "lg" }}
```

Comment:
The new design for embedding uses different font sizes for each card -- the SDK and EAJS title is larger than static and interactive.

Reply:
Hmmm, but why? Why can't they be same size? I kinda don't like having special style for certain element as everything should conform to our design system.

Reply:
@WiNloSt Not sure, this is a question for @AlessioLaiso:

<img width="300" alt="CleanShot 2568-10-24 at 16 44 47@2x" src="https://github.com/user-attachments/assets/ddb9cd87-85be-401a-8b61-4948777ef91c" />

<img width="300" alt="CleanShot 2568-10-24 at 16 44 39@2x" src="https://github.com/user-attachments/assets/b0b933cf-6472-4ae5-b939-5feefdcfae92" />